custom_components/google_fit/api.py

```diff
@@ -95,10 +95,13 @@ def __init__(self):
             lastUpdate=datetime.now(),
             activeMinutes=None,
             calories=None,
+            basalMetabolicRate=None,
             distance=None,
             heartMinutes=None,
             height=None,
             weight=None,
+            fat=None,
+            temperature=None,
```

Comment:
```suggestion
            bodyTemperature=None,
```
Similarly for body temperature